app/controllers/saml_idp_controller.rb

```diff
@@ -13,8 +13,8 @@ class SamlIdpController < ApplicationController
   include BillableEventTrackable
   include SecureHeadersConcern
 
-  prepend_before_action :skip_session_load, only: [:metadata, :remotelogout]
-  prepend_before_action :skip_session_expiration, only: [:metadata, :remotelogout]
+  prepend_before_action :skip_session_load, only: :metadata
```

Comment:
I think these are still good to have, they don't actually prevent us from loading the session, they just minimize some analytics logging busywork

Reply:
ah ok, wasn't sure if it was worth adding - will put them back